elastic/apm-server
# processor/otel: record array attributes as labels

## Motivation/summary

Handle array-type OpenTelemetry span attributes. Array attributes must be homogenous, so just store the values as `[]interface{}` and we'll record them as array fields in Elasticsearch.

## Checklist

~- [ ] Documentation has been updated~

## How to test these changes

Instrument an application with OpenTelemetry, recording an array-type attribute (ideally multiple, one for each primitive element type). Send to APM Server. Check the docs in Elasticsearch, ensure the attributes are recorded as labels, with the field value being an array of whatever the primitive field type is (array of keywords, array of bools, etc.)

## Related issues

Closes #5250
